tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md

```diff
@@ -28,116 +28,117 @@ The period for idle time for Factory accounts is different than for trial accoun
 
 [ACCORDION-BEGIN [Step 1: ](Create new SAP Fiori project)]
 
-1. In the **Welcome Page** click **Create project from a template**.
+1. In the **Welcome Page** click **Start from template**.
 
-    !![Open Dev Space](03-01-02 AppStudio Welcome Tab withOUT Extensions Loaded Notification_.jpg)
+    !![Open Dev Space](AppStudio Welcome Tab withOUT Extensions Loaded Notification_.jpg)
 
-    >If the Welcome Page does not appear, in the menu bar, select **View | Find Command** to open the **command palette** and select the command **SAP Business Application Studio: Welcome Page**. The command palette is opened at the top-center of the SAP Business Application Studio window.
+    >If the Welcome Page does not appear, in the menu bar, select **View | Find Command** to open the **command palette** and select the command **Welcome**. The command palette is opened at the top-center of the SAP Business Application Studio window.
 
-    >!![Welcome Page from command palette](03-01-03 AppStudio Welcome Page from Command Palette_.jpg)
+    >!![Welcome Page from command palette](AppStudio Welcome Page from Command Palette_.jpg)
 
     >The easiest way to develop an SAP Fiori freestyle app from scratch is to create it from a template. To continue developing an existing application, the best practice is to use git source code management and clone the repository.
 
-    >Using the UI wizard you can at any point click the `reset` button to reset the wizard at the top-right of the wizard screen, click the `Back` button to go back to the previous step, or click the specific wizard step to go back to that step.
-    >For convenience, click the **Explorer** view button to close the `Explorer` view.
+    >Using the UI wizard you can at any point click the `Back` button to go back to the previous step, or click the specific wizard step to go back to that step.
 
     > You can also create a project from the terminal using Yeoman.
 
-2. Make sure that the target folder is `/home/user/projects`, select the **SAP Fiori Freestyle Project** template, and click **Next**.
+2. For convenience, click the **Explorer** view button to close the `Explorer` view.
 
-    !![Fiori project template](03-03 AppStudio Fiori Project Template_.jpg)
+3. Make sure that the target folder is `/home/user/projects`, select the **SAP Fiori Freestyle Project** template, and click **Start**.
 
-3. For **Target Running Environment**, select the following, and click **Next**.
+    !![Fiori project template](AppStudio Fiori Project Template_.jpg)
```

Comment:
Please adjust your file names to match the formattin shown in the [style guide.](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtvf3pb64)
Also, please use the recommended file type PNG.

Reply:
done